app/views/about/hiring.html.haml

```diff
@@ -0,0 +1,86 @@
+= render ViewComponents::AboutNav.new(:hiring)
+
+#page-hiring
+  %header
+    .lg-container.c-shapes
+      = graphical_icon "exercism-face"
+      %h1.text-h1.mb-12 Exercism is hiring!
+      %p.text-p-2xlarge Come join a passionate team making programming education fun and effective. Here are some of the roles we’re hiring for.
+
+  .roles.sm-container
+    = link_to hiring_1_about_path, class: 'role' do
+      = graphical_icon "hiring-fundraiser", category: 'graphics', css_class: 'graphic'
+      .information
+        %h2.text-h3 Head of Partnerships and Fundraising
+        .remote-tag
+          = graphical_icon "stars"
+          100% remote
+        %p.text-p-large
+          %strong Can you build and execute a fundraising strategy?
+          We're looking for someone to develop sponsorship and partnership revenue to support our growth as we strive to become the leading resource for lifelong programming education.
+        .details
+          .salary Salary: £50k (~$68k, ~€60k).
+          .remote Remote
+
+    = link_to hiring_2_about_path, class: 'role' do
+      = graphical_icon "hiring-content", category: 'graphics', css_class: 'graphic'
+      .information
+        %h2.text-h3 Content Factotum
+        .remote-tag
+          = graphical_icon "stars"
+          100% remote
+        %p.text-p-large
+          %strong Passionate about programming education?
+          We're looking for someone to lead the content side of our platform, working with our community to create world-class resources. A varied and interesting role that mixes strategy, authoring and editing.
+        .details
+          .salary Salary: £50k (~$68k, ~€60k).
+          .remote Remote
```

Comment:
These two descriptions are now slightly different than the two newer ones. The latter have a series of questions followed by "Then you might be the perfect person to become our first full-time X!" Not necessary to change of course, but it is slightly inconsistent.